src/screens/SurveyDetail/question.js

```diff
@@ -31,7 +18,7 @@ const Questions = () => {
         <div className="questions__details">
           <div className="questions__number">1/5</div>
           <h1 className="questions__title"> { question }</h1>
-          <WheelSelect data={ data } multipleSelect={ true } />
+          <ThumbRating initialRating={ 3 } ratingCount={ 5 } ratingEmoji="smiley" />
```

Comment:
ThumbRating or Rating?

Reply:
Fixed in [bd3473c](https://github.com/rafayet-monon/surveyor/pull/20/commits/bd3473c1311399fa44adeeeed584af06d88500b2)